src/main/java/core/basesyntax/BinaryString.java

```diff
@@ -7,6 +7,17 @@
      * а возвращает String с представлением этого числа в двоичном виде.
      */
     public String toBinaryString(int value) {
-        return null;
+        StringBuilder sb = new StringBuilder();
+
+        if (value == 0) {
+            return "0";
+        }
+        for (;value != 0;) {
```

Comment:
looks like `while` in a `for` :)
```suggestion
        while (value != 0) {
```